logic/tool/TypeResolver.java

```diff
@@ -86,10 +102,10 @@ public Conjunction resolveLabels(Conjunction conjunction) {
         return conjunction;
     }
 
-    public Conjunction resolve(Conjunction conjunction) {
+    public Conjunction resolve(Conjunction conjunction, boolean insertable) {
         resolveLabels(conjunction);
-        TraversalBuilder traversalConstructor = new TraversalBuilder(conjunction, conceptMgr);
-        Map<Reference, Set<Label>> resolvedLabels = executeResolverTraversals(traversalConstructor);
+        TraversalBuilder traversalBuilder = new TraversalBuilder(conjunction, conceptMgr, insertable);
```

Comment:
Changed to `traversalBuilder` to reflect the class now being `TraversalBuilder`

Reply:
v good, consistency is great